src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -233,8 +234,9 @@ Status MutableObjectManager::WriteAcquire(const ObjectID &object_id,
   RAY_RETURN_NOT_OK(object->header->CheckHasError());
   RAY_CHECK(!channel->written) << "You must call WriteRelease() before WriteAcquire()";
 
-  RAY_RETURN_NOT_OK(
-      object->header->WriteAcquire(sem, data_size, metadata_size, num_readers));
+  auto timeout_point = ToTimeoutPoint(timeout_ms);
```

Comment:
Please use the type instead of `auto`

Reply:
Can you clarify a bit why is it preferred? With `auto` it is more readable here.